scripts/classes/buildings/Hearth.js

```diff
@@ -7,6 +7,8 @@ class Hearth extends Building {
    * Run on each tick.
    */
   tick() {
+    let hearthsOwned = curCiv.hearths.owned;
+    civData.wood -= hearthsOwned;
```

Comment:
Correct way to get amount of wood is

    civData.wood.owned

Reply:
Thanks! I was so close. 😉